### ISSUE TITLE
[Accessibility]A11y_VSCode_StatusBar_Keyboard: Using keyboard keys the StatusBar is not accessible

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51985]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : MAC OS HighSierra10.13.5

**Additional Details:**
Keyboard,MAS2.1.1,MAS2.4.3

**Repro Steps:**
1.Navigate to "Status Bar" at bottom of the screen after login.
2.Try to navigate "Status Bar" through the keyboard key such as "TAB".
3.Observed that,Using tab key "Status Bar" is not accessible.

**Actual:**
Using mouse only user can access the item list of "StatusBar".Using keyboard keys such as "tab,arrow keys" the statusBar is not accessible to user.

**Expected:**
Without using mouse user should be able to access the item of "StatusBar".Using "Tab and arrow keys" user should be able to access all the item list of "Status Bar".

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx](url)

**User Impact:**
Only keyboard using user will not able to access the "StatusBar"

**MAS Reference:**
MAS2.1.1- [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](url)

MAS2.4.3- [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}](url)

**Attachment For Reference:**
[[Accessibility]A11y_VSCode_StatusBar_Keyboard.pptx](https://github.com/Microsoft/vscode/files/2105372/Accessibility.A11y_VSCode_StatusBar_Keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes
